CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.4 (2021-12-17)
+
+- Fixed the code block component to handle dynamic updates to the code input property. [#55](https://github.com/blackbaud/skyux-lib-code-block/pull/55)
```

Comment:
Hey @blackbaud-paulah , curious on the formatting for code block. The docs page refers to it as the "SKY UX code block component": https://docs.blackbaud.com/stache/components/code-block
Why the hyphen and code formatting ticks?

Also, shouldn't `code-input` should just be...
> `code` input
(note only code is surrounded by the ticks, because the name of the input is just "code")

Reply:
@Blackbaud-AlexKingman Hi Alex. Feel free to ignore. I looked at prior changelogs, and they formatted component names  as code (https://github.com/blackbaud/skyux-lib-code-block/blob/master/CHANGELOG.md#140-2019-03-27), which is why I suggested the change. I probably misunderstood...

Reply:
All good! I still think we should format the final `code` b/c its an input... I'll do that now.